spinners.json

```diff
@@ -870,5 +870,39 @@
 			"🌲", 
 			"🎄"
 		]
+	},
+	"blink": {
+		"interval": 75,
+		"frames": [
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(-)(-)", 
+			"(_)(_)"
+		]
+	},
+	"fish": {
+		"interval": 275,
+		"frames": [
+			">))'>", 
+			" >))'>", 
+			"  >))'>", 
+			"   >))'>", 
+			"    >))'>", 
+			"   <'((<", 
+			"  <'((<", 
+			" <'((<"
+		]
```

Comment:
This needs padding on the right side so it doesn't move the spinner text during animation.

I'm also not sure if this would be a good fish. It's not that clear it's a fish. I would never have guessed if I hadn't seen the name.